pallets/kensetsu/src/migrations.rs

```diff
@@ -174,3 +174,91 @@ pub mod stage_correction {
         }
     }
 }
+
+pub mod storage_add_total_collateral {
```

Comment:
I'm going to delete all the old migrations before going to prod and release with `V1`. That's why I don't use versions for pallet storage.